app/components/teachers/details/past_induction_periods_component.rb

```diff
@@ -26,16 +26,24 @@ def induction_period_programme(period)
       end
     end
 
-    def edit_link(period)
-      return unless enable_edit
-
-      helpers.govuk_link_to('Edit', helpers.edit_admin_teacher_induction_period_path(teacher_id: teacher.id, id: period.id), no_visited_state: true)
-    end
-
-    def delete_link(period)
-      return unless enable_edit
-
-      helpers.govuk_link_to('Delete', helpers.confirm_delete_admin_teacher_induction_period_path(teacher_id: teacher.id, id: period.id), method: :get, class: 'govuk-link--destructive', no_visited_state: true)
+    delegate :govuk_link_to, to: :helpers
+
+    def actions_for(period)
+      return [] unless enable_edit
+
+      [
+        govuk_link_to(
+          'Edit',
+          helpers.edit_admin_teacher_induction_period_path(teacher, period),
+          no_visited_state: true
+        ),
+        govuk_link_to(
+          'Delete',
+          helpers.confirm_delete_admin_teacher_induction_period_path(teacher, period),
+          class: 'govuk-link--destructive',
```

Comment:
Not sure what `govuk-link--destuctive` is doing here, can't see any mention of it in `govuk-frontend` 🤔